sfa_dash/form_utils/converters.py

```diff
@@ -588,7 +597,11 @@ def payload_to_formdata(cls, payload_dict):
         form_params['metrics'] = report_parameters.get('metrics', [])
         form_params['period-start'] = report_parameters['start']
         form_params['period-end'] = report_parameters['end']
-        form_params['costs'] = report_parameters['costs']
+        form_params['costs'] = report_parameters.get('costs', [])
```

Comment:
looks unrelated to this PR. that's ok, just double checking it's not a merge issue.